### DISCUSSION TITLE
Can't Delete Files

### DISCUSSION THREAD

**Discussion by ghost** [community/community#68041] [Repositories]
Whenever i am trying delete file it shows it deleted but when i refresh i coming back please help

**Comment by tiagoangelos** [comment]
To resolve the error, download the repository to your desktop, open the file manually select and hold: Ctrl+del and update your repository by sending the changes, I hope I helped!

**Comment by seyLu** [comment]
Hi @codeConquests,

Make sure after you delete the files, git add > commit > push the changes. Or if you are using Github web:

- go to file/directory you want to delete.
- click the three dots, and click `delete file`
- click `commit changes...`
- write a commit message
- click `commit changes`
- you should get an alert banner that the file was successfully deleted

![image](https://github.com/community/community/assets/98249191/6a4f14a1-343d-4f8f-97b1-75f1cca410ec)
![Screenshot from 2023-09-24 10-54-47](https://github.com/community/community/assets/98249191/0220b7ce-0682-40ed-8237-910e4c5f43ec)
![image](https://github.com/community/community/assets/98249191/9610426a-d4db-4983-b90e-9f63c36f964a)
![image](https://github.com/community/community/assets/98249191/15f27341-3ea2-412d-a3aa-971ec9acda84)

**Reply by ghost** [reply]
Thankyou so much, it works

**Reply by Elsa402** [reply]
it bothers me half an hour, now I know it's because I didnt click the 'commit change'button... Thank you!

**Comment by madromas** [comment]
I see no delete option, what is going on???

Update.... On the window where delete button WAS visible before, now you have to scroll down on that pop up menu window... Kinda lame...

![Screenshot 2024-08-20 194721](https://github.com/user-attachments/assets/8dc1d6ff-4fba-47e2-94b7-085de73e4318)

**Reply by minhp03** [reply]
I was looking for that so bad.!!! thank you
